__tests__/index.test.js

```diff
@@ -87,6 +87,7 @@ describe("/api/reviews/:review_id", () => {
           category: "euro game",
           owner: "mallionaire",
           created_at: "2021-01-18T10:00:20.514Z",
+          comment_count: 0,
```

Comment:
We shouldn't be changing past tests. This should be a new test by itself that doesn't cause previous ones to fail.